batchflow/models/torch/base.py

```diff
@@ -1793,6 +1810,50 @@ def load(self, file, make_infrastructure=False, mode='eval', pickle_module=dill,
         else:
             self._parse_devices()
 
+        if isinstance(file, str):
+            if fmt == "safetensors" or (fmt is None and file.endswith(".safetensors")):
+                from safetensors.torch import load_file
+                state_dict = load_file(file)
+
+                self.initialize()
```

Comment:
Why do you need it here? If you try to load a safetensors model using your function, you will get an error:
```model.save('tmp.safetensors', fmt='safetensors', pickle_metadata=False)``` -> `ValueError: Set "loss" in model configuration!`